gradle/formatting.gradle

```diff
@@ -127,6 +127,8 @@ def projectPathsToExclude = [
   ':x-pack:plugin:ccr',
   ':x-pack:plugin:ccr:qa',
   ':x-pack:plugin:core',
+  // TODO: REMOVE BEFORE MAKING PR REVIEWABLE:
```

Comment:
I will remove this before merging, this to make potential merge conflicts easier to resolve.